src/front/wgsl/lower/mod.rs

```diff
@@ -1146,11 +1162,7 @@ impl<'source, 'temp> Lowerer<'source, 'temp> {
 
                     let initializer = match v.init {
                         Some(init) => {
-                            let initializer =
-                                self.expression(init, ctx.as_expression(block, &mut emitter))?;
-                            ctx.as_expression(block, &mut emitter)
-                                .grow_types(initializer)?;
```

Comment:
Was this call just completely extraneous?

Reply:
It was for the call a few lines below to `resolved_inner`.